README.md

```diff
@@ -1,10 +1,11 @@
 # indowiki
-This is a wiki for an indigenous knowledge and wisdom
 -------------------------------------------------------
 ### Lets retain our indiginious knowledge and wisdom
 ---------------------------------------------------
 Overview  / Back ground 
 
-[ ]  **What is Indiginious**
-[ ]  **Why indiginious**
-[ ]  **Main Catagories**
+- [ ] **What is Indiginious**
+- [ ] **Why indiginious**
+- [ ] **Main Catagories**
+
+<-- more detailed info will follow soon -->
```

Comment:
This section has now sort out properly.
nice one move on. Now the PR approval request is also properly functioning.
Me as an admin/creator of the given repository i can still merge the created pull with out waiting for any other approval.

Reply:
Now i have approved the merge